bin/generator.ps1

```diff
@@ -0,0 +1,21 @@
+
+write-host "DataHelix Generator Executor Script"
+
+$DATAHELIX_GENERATOR_HOME="c:\datahelix\generator"  # Change this to be the path that the `generator.jar` file is in
```

Comment:
I suggest these scripts are installed to the same directory, or a relative path to, the jar file. 

The user can then choose to install the files wherever they like, and put these scripts alongside. Would also help with the chocolatey install, not requiring a fixed directory path.

Reply:
I think the installer would update the path during install, as this is the only time we will know where the user has selected to put the files.